src/components/List/SimpleList/SimpleList.jsx

```diff
@@ -51,6 +51,12 @@ const propTypes = {
   pageSize: PropTypes.string,
   /** callback function returned a modified list */
   onListUpdated: PropTypes.func,
+  /** optionally renders SimplePagination at the bottom of the list */
```

Comment:
Separate PR opened for this fix: https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1692